app/packages/looker-3d/src/action-bar/slice-selector.tsx

```diff
@@ -0,0 +1,74 @@
+import { Selector } from "@fiftyone/components";
+import { SelectorProps } from "@fiftyone/components/src/components/Selector/Selector";
+import * as fos from "@fiftyone/state";
+import { useCallback, useMemo } from "react";
+import { useRecoilState } from "recoil";
+
+interface SliceSelectorProps {
+  dataset: fos.State.Dataset;
+}
+
+const SliceSelectorComponent = ({
+  value,
+  className,
+}: {
+  value: string;
+  className: string;
+}) => {
+  return <div className={className}>{value}</div>;
+};
+
+export const SliceSelector = ({ dataset }: SliceSelectorProps) => {
```

Comment:
This seems to be a bit redundant to:

<img width="231" alt="image" src="https://user-images.githubusercontent.com/462228/220970701-8bb91303-d6f3-48af-a546-2f1acd61e92c.png">